src/styles/components/_syntax-highlighter.scss

```diff
@@ -161,6 +165,11 @@ pre > code.diff-highlight .token.inserted:not(.prefix) {
             word-wrap: normal;
             tab-size: 4;
             hyphens: none;
+
+            @media (max-width: 300px) {
```

Comment:
Please define this `300px` in css custom properties to allow consumer to update it without updating the whole `mynah-ui`. This 300px might not be applicable for every case. 

Additionally, we're trying avoid using static values for all cases. If it is not mandatory.